src/Utopia/Messaging/Adapter/SMS/Msg91.php

```diff
@@ -3,22 +3,23 @@
 namespace Utopia\Messaging\Adapter\SMS;
 
 use Utopia\Messaging\Adapter\SMS as SMSAdapter;
-use Utopia\Messaging\Messages\SMS;
+use Utopia\Messaging\Messages\SMS as SMSMessage;
+use Utopia\Messaging\Response;
 
 // Reference Material
 // https://docs.msg91.com/p/tf9GTextN/e/7WESqQ4RLu/MSG91
 
 class Msg91 extends SMSAdapter
 {
-    private string $templateId;
-
```

Comment:
We can't remove this or the setter, it's needed to make the request

Reply:
This has been moved to constructor. we don't need the setter as it's being set in the constructor now

Reply:
Please double check that it won't cause issues with this PR https://github.com/appwrite/appwrite/pull/7097